nose/case.py

```diff
@@ -357,7 +359,7 @@ def __str__(self):
         name = "%s.%s.%s" % (self.cls.__module__,
                              self.cls.__name__,
                              name)
-        if arg:
+        if not arg == '()':
```

Comment:
I'm not sure this is doing what you think it is.  It's comparing `arg` (a tuple) against a string, which doesn't seem correct.